.env.example

```diff
@@ -1,2 +1,3 @@
 NEXT_PUBLIC_NETWORK_ID=mainnet
-NEXT_PUBLIC_HOSTNAME=https://near.org
\ No newline at end of file
+NEXT_PUBLIC_HOSTNAME=https://near.org
+NEXT_PUBLIC_PAGE_URLS=
```

Comment:
All we need here is to pass string of pages we need
`'https://beta.near.org, https://beta.near.org/ecosystem, https://beta.near.org/use'` to generate .html files for them

Reply:
Please move this config into `package.json` so we have the options for this process centralized in one place. See the `main` branch as an example of a similar config we're using for react-snap.